homeworks/taridaniel/3_conditionals_loops/flat_finder.py

```diff
@@ -0,0 +1,13 @@
+city = input("Please enter an American city: ").title()
+price = int(input("Please enter the price (USD): "))
+
+if (city == "New York" or city == "San Francisco") and price < 4000:
+    print(f"I like these conditions in {city}, I want to live here")
+elif city == "Washington": 
+    print(f"I hate {city}, I don't want to live in this city")
+elif city == "Chicago":
+    print(f"I love {city} so much! The price of the rent doesn't matter, I want to live here!")
+elif (city != "New York" or city != "San Francisco" or city != "Washington" or city != "Chicago") and price <= 3000:
```

Comment:
`(city != "New York" or city != "San Francisco" or city != "Washington" or city != "Chicago") and`

Ez a rész ebbe a feltételbe már nem kell. Ha idáig eljut a végrehajtás, akkor biztosak lehetünk benne hogy a city nem ezen városok valamelyike, mert előtte már mindegyiket megvizsgáltad. Elég annyi hogy ha az árat megvizsgálod hogy 3000 alatt van e.